guide/03-the-gis/offline-cloning-backups.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<ul><li>It might help to provide a sentence or 2 providing guidance as to what to do with those error messages, or to at least say "you can troubleshoot reason why these items didn't import". The ending of the document felt very abrupt and kind of open ended to me.</li><li>I think it would help to provide a conclusion paragraph as well.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>